java-client/src/generated/java/org/opensearch/client/opensearch/_types/mapping/SourceFieldMode.java

```diff
@@ -30,20 +30,36 @@
  * GitHub history for details.
  */
 
-package org.opensearch.client.opensearch.indices;
+//----------------------------------------------------
+// THIS CODE IS GENERATED. MANUAL EDITS WILL BE LOST.
+//----------------------------------------------------
 
-import org.opensearch.client.json.JsonpDeserializer;
+package org.opensearch.client.opensearch._types.mapping;
 
-// typedef: indices.exists.Response
+import javax.annotation.Generated;
+import org.opensearch.client.json.JsonEnum;
+import org.opensearch.client.json.JsonpDeserializable;
 
-public class ExistsResponse {
-    public ExistsResponse() {}
+// typedef: _types.mapping.SourceFieldMode
 
-    /**
-     * Singleton instance for {@link ExistsResponse}.
-     */
-    public static final ExistsResponse _INSTANCE = new ExistsResponse();
+@JsonpDeserializable
+@Generated("org.opensearch.client.codegen.CodeGenerator")
+public enum SourceFieldMode implements JsonEnum {
+    Disabled("disabled"),
 
-    public static final JsonpDeserializer<ExistsResponse> _DESERIALIZER = JsonpDeserializer.fixedValue(ExistsResponse._INSTANCE);
+    Stored("stored"),
 
+    Synthetic("synthetic");
```

Comment:
I think we don't support `synthetic` either

Reply:
Yeah, looked into it and this enum and the associated `mode` property on `SourceField` don't exist in OS.

Have made a PR removing it from the spec (https://github.com/opensearch-project/opensearch-api-specification/pull/652) and removed it here.